webidls/Findable.webidl

```diff
@@ -0,0 +1,28 @@
+enum FindCaseSensitivity {
+  "case-sensitive",
+  "case-insensitive"
+};
+
+enum StopFindAction {
+  "clearSelection", // Clear the selection
+  "keepSelection", // Translate the selection into a normal selection.
+  "activateSelection", // Focus and click the selection node
+};
+
+interface Findable {
+  Promise<void> setFindableHandler(FindableHandler Handler);
+
+  Promise<void> find(DOMString text, FindCaseSensitivity caseSensitivity);
```

Comment:
I suspect one may want to know what is actually found like actually matching `text` or `node.outerHTML`.